STL_Extension/doc/STL_Extension/CGAL/algorithm.h

```diff
@@ -7,7 +7,7 @@ namespace CGAL {
 /*!
 \ingroup STLAlgos
 
-\deprecated This function is deprecated, std::copy_n should be
+\deprecated This function is deprecated, `std::copy_n()` should be
```

Comment:
I question that choice, and all the similar changes. In C++ `std::copy_n` designs something: that is an overload set, documented: https://en.cppreference.com/w/cpp/algorithm/copy_n

But `std::copy_y()` does not design anything. In my opinion that is an error.

Reply:
Don't we have the documentation convention to put `()`  behind functions in the manual?  [link](https://github.com/CGAL/cgal/wiki/Writing-Documentation#function-documentation).   It is well possible that I wrote this.

Reply:
It might be a consequence of the behavior of Doxygen with auto-links. Doxygen will [generate a link (see doc)](https://doxygen.nl/manual/autolink.html) for:
  - `convex_hull_2()`, or
  - `CGAL::convex_hull_2`,

but not for `convex_hull_2` without `()` or `CGAL::`. Maybe at the time we did not want to add `CGAL::` everywhere, and decided to use something like `convex_hull_2()`, even if that is not correct.

As a matter of fact, in [our wiki](https://github.com/CGAL/cgal/wiki/Writing-Documentation#function-documentation) I even would like to change ``The function `convex_hull_2()` ....`` to ``The function template `CGAL::convex_hull_2` ....`` to identify `CGAL::convex_hull_2` as a function **template**.